### DISCUSSION TITLE
Help answering questions on stackoverflow

### DISCUSSION THREAD

**Discussion by WolfgangFahl** [jupyter/nbgrader#1632] [General]
### Expected behavior
Questions on stackoverflow https://stackoverflow.com/questions/tagged/nbgrader should be answered and this support
option be mentioned e.g. in the readme
### Actual behavior
nbgrader tag was just created to day
### Steps to reproduce the behavior

visit https://github.com/jupyter/nbgrader and search for "stackoverflow"

**Comment by perllaghu** [comment]
Feel free to answer questions on StackOverflow

**Comment by WolfgangFahl** [comment]
@perllaghu i already do. This is not really an issue - since #1627  is open currently discusions are labeled "discussion". The only change this ticket might have is changing the readme by adding a link to the stackoverflow https://stackoverflow.com/questions/tagged/nbgrader and may be the https://stackoverflow.com/tags/nbgrader/info which currently is pending since the tag is new.

**Comment by jhamrick** [comment]
Thanks for creating the tag on stackoverflow @WolfgangFahl ! I'd be a bit hesitant though I add it to the readme, just because I think we're already stretched thin answering questions on the github repo---so unless someone commits to being in charge of answering stackoverflow questions I wouldn't want to give the impression that it's a good venue for getting a response. I unfortunately don't think there is bandwidth for it to be an official support channel.

**Comment by WolfgangFahl** [comment]
At the current frequency of one question every two month i don't see a problem to look after these questions myself. I just assume that helping people on stackoverflow with their needs will increase the popularity and value of nbgrader.
